ui/src/app/pages/data/data-component.tsx

```diff
@@ -99,14 +99,8 @@ export const DataComponent = withCurrentWorkspace()((props: Props) => {
   const loadResources = async() => {
     try {
       setIsLoading(true);
-      setResourceList(
-        (await workspacesApi().getWorkspaceResources(workspace.namespace, workspace.id, {typesToFetch: resourceTypesToFetch}))
-          .map(result => {
-            return {
-              ...result,
-              // TODO (RW-4682): Fix this nonsense
```

Comment:
fixed that nonsense

Reply:
RW-4682 will be complete after we've confirmed the **whole codebase** is free of such nonsense.